Failed to parse: Got unexpected unicode

I'm getting an error parsing a few of our files like `error: Failed to parse t.py: Got unexpected unicode at line 1 column 4`.

## `t.py`

```python
"\N{BACKSPACE}another cool trick"
```

## output

```
❯ ruff --isolated t.py
error: Failed to parse t.py: Got unexpected unicode at line 1 column 4
t.py:1:5: E999 SyntaxError: Got unexpected unicode
Found 1 error.
```

## version

```
ruff 0.0.252
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -14,8 +14,8 @@ libcst = { git = "https://github.com/charliermarsh/LibCST", rev = "80e4c1399f95e
 once_cell = { version = "1.16.0" }
 regex = { version = "1.6.0" }
 rustc-hash = { version = "1.1.0" }
-rustpython-common = { git = "https://github.com/RustPython/RustPython.git", rev = "351d464448607413dc12de6905f1165f8d45af54" }
-rustpython-parser = { features = ["lalrpop"], git = "https://github.com/RustPython/RustPython.git", rev = "351d464448607413dc12de6905f1165f8d45af54" }
+rustpython-common = { git = "https://github.com/RustPython/RustPython.git", rev = "aa8336ee94492b52458ed8e1517238e5c6c2914c" }
+rustpython-parser = { features = ["lalrpop"], git = "https://github.com/RustPython/RustPython.git", rev = "aa8336ee94492b52458ed8e1517238e5c6c2914c" }
 schemars = { version = "0.8.11" }
 serde = { version = "1.0.147", features = ["derive"] }
 serde_json = { version = "1.0.87" }

